### PR TITLE
build: update prettier to 2.7.1

### PR DESCRIPTION
**Pull request by dajohi**
See https://prettier.io/blog/2021/05/09/2.3.0.html

### CODE REVIEW

**Review by bgptr — APPROVED** [review]
Looks pretty good

**Comment by alexlyp** [comment]
Looks good to me, but due to it touching so many files and so many PRs outstanding we'll wait on this until the bulk of the currect PRs are merged then the new prettifier can be run again on top of those.

**Review by alexlyp — APPROVED** [review]

**Pull request merged**
